### PR TITLE
[Merged by Bors] - feat: Port CategoryTheory.Adjunction.Whiskering

### PR DESCRIPTION
**Pull request by casavaca**
Rename and reflow only.

---

- [x] depends on: #2198 
[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#2198
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#2198~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

File: Mathlib/CategoryTheory/Adjunction/Whiskering.lean [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+-/
+
+
+namespace CategoryTheory.Adjunction
+
+open CategoryTheory
+
+variable (C : Type _) {D E : Type _} [Category C] [Category D] [Category E] {F : D ⥤ E} {G : E ⥤ D}
+
+--  `tidy` works for all the proofs in this definition, but it's fairly slow.
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Adjunction/Whiskering.lean:29]: This doesn't seem relevant anymore

File: Mathlib/CategoryTheory/Adjunction/Whiskering.lean [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      counit :=
+        { app := fun X =>
+            (Functor.associator _ _ _).hom ≫ whiskerLeft X adj.counit ≫ (Functor.rightUnitor _).hom
+          naturality := by intros; ext; dsimp; simp }
+      left_triangle  := by ext; dsimp; simp
+      right_triangle := by ext; dsimp; simp
+    }
+#align category_theory.adjunction.whisker_right CategoryTheory.Adjunction.whiskerRight
+
+-- `tidy` gets stuck for `left_triangle'` and `right_triangle'`.
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Adjunction/Whiskering.lean:49]: Idem

**Comment by jcommelin** [comment]
I suggest simply removing those comments about tidy, instead of trying to rewrite them somehow.

bors d+

**Pull request closed without merging**